projects/components/src/utils/id-generator/id-generator.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+/**
+ * Object containing the logic to generate unique identifiers whenever it's {@link IdGenerator#id} method is called
```

Comment:
Object containing is redundant. Please avoid words like class, object, interface, logic, code, called in documentation.

Generates unique IDs.